docs/source/concepts/architecture/cached-files.rst

```diff
@@ -0,0 +1,25 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is subject to the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+Cache Files Managed by GeoIPS
+*****************************
+GeoIPS manages a number of different cache files. These are produced to speed
+up different processes such as the CLI and plugin loading. These cache files
+are managed by GeoIPS and should not be modified or deleted by the user. They
+are stored in the ``$GEOIPS_CACHE_DIR`` directory, which is set by the
+``GEOIPS_CACHE_DIR`` environment variable. If this variable is not set, the
```

Comment:
Why are my conversations not blocking merge ?

Reply:
Please review my other conversation before merging.

Reply:
It looks like they're blocking now.